app/core/store/modules/products.js

```diff
@@ -59,6 +75,21 @@ export default {
       return productsCollection.getBasicAnnualSubscriptionForUser(window.me).toJSON()
     },
 
+    // Returns price of basic annual subscription modified by coupon
+    basicAnnualSubscriptionPrice (_state, getters) {
+      if (!getters.basicAnnualSubscriptionForCurrentUser || !getters.activeCouponID) {
+        // Default price of product. Can safely show while loading.
+        return 99
+      }
+
+      const coupon = _.find(getters.basicAnnualSubscriptionForCurrentUser.coupons, { code: getters.activeCouponID })
+      if (coupon && typeof coupon.amount === 'number') {
```

Comment:
I tested it on my local without signup and adding coupon=brazil. It didn't work for me because coupon had following data:
```
amount: "3960"
code: "brazil-annual"
```
Amount was string instead of number, not sure why that happened to me, any idea?

Reply:
I think I did a "quick" optimization and broke it. Should be resolved now! Thank you!